### PR TITLE
Support priority class for tidb cluster

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve? 
https://github.com/pingcap/tidb-operator/issues/910

### What is changed and how does it work?

- `tidb-cluster` chart template
-  each componenet member manager
- `PodAttributesSpec` in `pkg/apis/pingcap.com/v1alpha1/types.go`

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Code changes

 - Has Helm charts change
 - Has Go code change

Side effects

 - N/A

Related changes

 - N/A

### Does this PR introduce a user-facing change?:
NONE

### CODE REVIEW

**Comment by Yisaer** [comment]
./run-e2e-in-kind

**Comment by Yisaer** [comment]
./run-e2e-in-kind

File: charts/tidb-cluster/values.yaml [hunk 1]
```diff
@@ -252,6 +256,10 @@ tikv:
   # refer to https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
   podSecurityContext: {}
 
+  # Specify the priorityClassName for TiKV Pod.
+  # refer to https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#how-to-use-priority-and-preemption
+  priorityClassName:
```
Comment (weekface) [inline charts/tidb-cluster/values.yaml]:
```suggestion
  priorityClassName: ""
```

?

**Comment by Yisaer** [comment]
./run-e2e-in-kind

**Pull request merged**